ui/admin/app/components/form/field/list-wrapper/select/index.hbs

```diff
@@ -0,0 +1,81 @@
+<Hds::Table class='list-wrapper-field' name={{@name}}>
+
+  <:body as |B|>
+
+    {{#each @options as |select index|}}
+      <B.Tr>
+        <B.Td>
+          <Hds::Form::Select::Base
+            @value={{select.value}}
+            @width={{@width}}
+            disabled={{@disabled}}
+            aria-label={{t 'titles.value'}}
+            {{on 'change' (set-from-event select 'value')}}
+            as |F|
+          >
+            <F.Options>
+              {{#each @selectOptions as |selectOption|}}
+                <option
+                  value={{selectOption}}
+                  selected={{eq selectOption select.value}}
+                >
+                  {{selectOption}}
+                </option>
+              {{/each}}
+            </F.Options>
+          </Hds::Form::Select::Base>
+        </B.Td>
+
+        <B.Td>
+          {{#if this.removeOptionByIndex}}
+            <Hds::Button
+              data-test-remove-option-button={{select.value}}
+              @text={{t 'actions.remove'}}
+              @color='critical'
+              @icon='trash'
+              @isIconOnly={{true}}
+              disabled={{@disabled}}
+              {{on 'click' (fn this.removeOptionByIndex index)}}
+            />
+          {{/if}}
+        </B.Td>
+      </B.Tr>
+    {{/each}}
+
+    <B.Tr>
+      <B.Td>
+        <Hds::Form::Select::Base
+          @value={{this.newOptionValue}}
+          @width={{@width}}
+          disabled={{@disabled}}
+          aria-label={{t 'titles.value'}}
+          {{on 'change' (set-from-event this 'newOptionValue')}}
+          as |F|
+        >
+          <F.Options>
+            <option value=''></option>
```

Comment:
It should still say "choose a value", as well as not be selectable by the user, we have examples in the codebase

Reply:
@ZedLi thank you found that reference code.  Incoming on next commit